.appveyor.yml

```diff
@@ -67,9 +67,6 @@ test_script:
       }
   - npm run coverage:system-tests
 
-after_test:
```

Comment:
Let's save some cycles off AppVeyor since we are also doing this on CircleCI for now.